### DISCUSSION TITLE
Blue Cube Compatibility

### DISCUSSION THREAD

**Discussion by TheRealKush1** [darthcloud/BlueRetro#713] [General]
I'm working on making a OG gamecube controller bluetooth. Just wondering if I used the blue cube firmware designed by Nathan reeves if it would be compatible. Not sure if his code will even work without his product so i might not even be able to use it anyways. Thanks. I appreciate your hard work on this project.

**Comment by Dreded** [comment]
im looking into this myself, in my untested opinion since the BlueCube would connect natively to a switch it should also connect to Blue Retro just fine as BlueRetro supports switch controllers.

**Reply by TheRealKush1** [reply]
Didn't see this till now. So not sure if you already know but the code needs rewritten for bluecube mod. Both versions are outdated and will not compile on the current esp idf. I only know some python and not C so I haven't been able to do anything about rewriting it.
